### PR TITLE
fix: Fix cronjob status display err in detail page

### PR DESCRIPTION
**Pull request by moweiwei**
### What type of PR is this?

/kind bug

### Which issue(s) this PR fixes:

Fixes ##3053

### Special notes for reviewers:
```
```

### Does this PR introduced a user-facing change?

```release-note
fix: Fix cronjob status display err in detail page
```

### Additional documentation, usage docs, etc.:

```docs

```

/assign harrisonliu5

### CODE REVIEW

**Comment by harrisonliu5** [comment]
Thanks for your contribution!

File: src/pages/projects/containers/CronJobs/Detail/index.jsx [hunk 1]
```diff
@@ -128,7 +129,7 @@ export default class JobDetail extends React.Component {
   getAttrs = () => {
     const { detail } = this.store
     const { spec = {} } = detail
-    const status = this.suspend ? t('Suspend') : t('RUNNING')
+    const status = getCronJobStatus(detail) ? t('Suspend') : t('RUNNING')
```
Comment (harrisonliu5) [inline src/pages/projects/containers/CronJobs/Detail/index.jsx:132]: Maybe this status is not 'Running'.

image

Comment (moweiwei) [reply src/pages/projects/containers/CronJobs/Detail/index.jsx]: sorry, its my careless, please review again, @harrisonliu5

**Pull request merged**